work_queue/src/bindings/python3/work_queue.binding.py

```diff
@@ -2073,10 +2073,16 @@ class RemoteTask(Task):
     # @param coprocess  The name of the coprocess which has the function you wish to execute. The coprocess should have a name() method that returns this
     # @param
     # @param command    The shell command line to be exected by the task.
-    # @param kwargs	    keyword arguments used in function to be executed by task. An optional kwarg exec_method can be specified to one of fork, thread, and direct to choose between those three methods of execution
-    def __init__(self, fn, coprocess, **kwargs):
+    # @param args       positional arguments used in function to be executed by task. Can be mixed with kwargs
+    # @param kwargs	    keyword arguments used in function to be executed by task. An optional kwarg exec_method can be specified to one of fork, thread, and direct to choose between those three methods of execution. Another optional kwarg is work_queue_argument_dict, which contains a dictionary of keyword arguments for the remote function
+    def __init__(self, fn, coprocess, *args, **kwargs):
```

Comment:
What do you think of adding a `specify_exec_method()` to `RemoteTask`?   I don't like that the kwarg `exec_method` gets mixed with the args to the function.
Such method may also simplify the code needed to remove the `exec_method` key from the kwarg dict.